src/data/complex/exponential.lean

```diff
@@ -1290,6 +1298,63 @@ begin
     mul_le_mul_of_nonneg_left (sum_div_factorial_le _ _ hn) (pow_nonneg (abs_nonneg _) _)
 end
 
+lemma exp_bound' {x : ℂ} {n : ℕ} (hx : abs x / (n.succ) ≤ 1 / 2) :
+  abs (exp x - ∑ m in range n, x ^ m / m!) ≤ abs x ^ n / (n!) * 2 :=
+begin
+  rw [← lim_const (∑ m in range n, _), exp, sub_eq_add_neg, ← lim_neg, lim_add, ← lim_abs],
+  refine lim_le (cau_seq.le_of_exists ⟨n, λ j hj, _⟩),
+  simp_rw ← sub_eq_add_neg,
+  show abs (∑ m in range j, x ^ m / m! - ∑ m in range n, x ^ m / m!)
+    ≤ abs x ^ n / (n!) * 2,
+  let k := j - n,
+  have hj : j = n + k,
+    exact (nat.add_sub_of_le hj).symm,
+  rw hj,
+  rw sum_range_add_sub_sum_range,
+  calc abs (∑ (k : ℕ) in range k, x ^ (n + k) / ((n + k)! : ℂ))
+      ≤ ∑ (k : ℕ) in range k, abs (x ^ (n + k) / ((n + k)! : ℂ)) : abv_sum_le_sum_abv _ _
+  ... ≤ ∑ (k : ℕ) in range k, (abs x) ^ (n + k) / (n + k)! :
+          begin
+            refine sum_le_sum (λ m hm, _),
+            simp only [complex.abs_cast_nat, complex.abs_div],
+            rw [abv_pow abs],
+          end
+  ... ≤ ∑ (k : ℕ) in range k, (abs x) ^ (n + k) / (n! * n.succ ^ k) :
+          begin
+            refine sum_le_sum (λ m hm, _),
+            apply div_le_div (pow_nonneg (abs_nonneg x) (n + m)) (le_refl _),
+            apply mul_pos,
+            rw nat.cast_pos,
+            exact nat.factorial_pos n,
+            apply pow_pos,
+            rw nat.cast_pos,
+            exact nat.succ_pos n,
+            rw [<-nat.cast_pow, <-nat.cast_mul, nat.cast_le],
+            exact (nat.factorial_mul_pow_le_factorial),
```

Comment:
Can you use braces to delimit the branches of the proof?

Reply:
done